app/views/users/show.html.erb

```diff
@@ -1,12 +1,16 @@
-<h1><%= @user.name %></h1>
-<ul>
-  <% @user.posts.each do |post| %>
-    <%= link_to user_post_path(@user, post.id) do %>
-      <li>
-        <h3><%= post.title %> __ post.id = <%= post.id %></h3>
-        <p><%= post.text %></p>
-      </li>
+<div class="flex flex-col w-full gap-3">
+  <%= render partial:'shared/user', locals: {user: @user} %>
+  <div class="flex flex-col gap-3 w-full p-4 border border-black border-solid">
+    <h3>Bio</h3>
+    <p><%= @user.bio %></p>
+  </div>
+  <ul class="flex flex-col gap-3 w-full">
+    <p class="only:block hidden">There are no posts yet!</p>
+    <% @user.last_3_posts.each do |post| %>
+      <%= link_to user_post_path(@user, post.id) do %>
+        <%= render partial:'shared/post_highlight', locals: {user: @user, post: post} %>
+      <% end %>
     <% end %>
-  <% end %>
-</ul>
-<button>Show all posts</button>
\ No newline at end of file
+  </ul>
+  <button>Show all posts</button>
+</div>
```

Comment:
- [ ] Kindly ensure this button is active and when clicked takes the user to the post index page,
![image](https://user-images.githubusercontent.com/61408860/200675559-64c7e630-04f8-4c63-9afd-fb6c0932b5b5.png)

Reply:
### :heavy_check_mark: Changes applied

Thanks for pointing that out! :smile: